### ISSUE TITLE
extra indent in Python Interactive window

### ISSUE THREAD

**Issue by peterbandi** [Microsoft/PTVS#1054]
Hi,

The Python Interactive window adds an extra space to indentation upon pressing Enter. For example this is what I get if I try to type in a function and simply press Enter after each row:
```
In [65]: def foo():
   ....: 	 print 1
   ....: 	  print 2
   ....: 	   print 3
   ....: 		print 4
   ....: 		 print 5
   ....: 		  print 6
   ....: 		   print 7
```
I have set Tabs for indentation in Tools/Options/Text Editor/Python/Tabs:
* Indenting = Smart
* Tab size = 4
* Indent size = 4
* Keep tabs

I am using Visual Studio Community 2015 version 14.0.24720.00 Update 1 with Python Tools for Visual Studio version 2.2.31105.00

**Comment by zooba** [comment]
While it certainly does some weird indentation in the interactive window, I can't get the same behavior you're seeing.

Could you find the following files below in File Explorer, right-click and view Properties, and copy the version number from the Details tab for me? (You may need to adjust the directories, but if you do them I'm sure you'll know how.)

```
C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\PrivateAssemblies\Microsoft.VisualStudio.InteractiveWindow.dll
C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\PrivateAssemblies\Microsoft.VisualStudio.VsInteractiveWindow.dll
```

I expect them to start with `1.1`, but if they start with `1.0` then your Update 1 installation hasn't worked properly and this strange behavior may be reflecting that.

**Comment by peterbandi** [comment]
The version of those files are starting with 1.1. I had this behavior during my entire workday that is why I created the ticket. I tried to change the settings I wrote. Than it was either the described erroneous behavior or it did not indented at all (which was also not good).

Lately I had to restart Visual Studio and now the Interactive Window works fine. I cannot reproduce the error any more. I have no idea what caused it previously. Sorry for the false alarm.

**Comment by zooba** [comment]
It's not entirely a false alarm. I'll close this issue, but when it comes up again we'll know that it happens intermittently and hopefully will be able to get some more information.

Thanks!

**Comment by peterbandi** [comment]
It is happening again. Do you need any information that I can collect?

**Comment by zooba** [comment]
Right now the only thing I can think of is what you were doing leading up to it happening? Did it work and then stop working? Or was it behaving like that from right when you started VS?

**Comment by peterbandi** [comment]
It worked fine for a while. I have started Visual Studio on Monday and worked on a python project ever since. I have not exited it at the end of the day because a test was running, so I kept the computer turned on. Then today by the end of the day it started to show the erroneous behavior again. I checked the history of the interactive window but there was nothing before the reoccurence of the error that I haven't done at least a few times since I started Visual Studio. After I restarted the Visual Studio the interactive window worked fine again.
